spec/views/events/participants.html.erb_spec.rb

```diff
@@ -52,7 +52,8 @@
   it "displays print badges button (in event execution phase)" do
     @event.published = true
     @event.application_deadline = Date.yesterday
-    @event.application_status_locked = true
+    @event.acceptances_have_been_sent = true
```

Comment:
In this case we should use indeed a factory for an event in execution phase (it is even in the title of the test) :)